ctests/test_kernels.cpp

```diff
@@ -19,7 +19,7 @@ class KernelTest : public StructureTest {
 
 };
 
-using KernelTypes = ::testing::Types<SquaredExponential, NormalizedDotProduct>;
+using KernelTypes = ::testing::Types<SquaredExponential, NormalizedDotProduct, DotProduct>;
```

Comment:
should it be named as DotProduct or NonNormalizedDotProduct?

Reply:
I think `DotProduct` is fine (and shorter), since it is the "pristine" dot product <di, dj>. While the `NormalizedDotProduct` is the normalized version <di, dj>/(|di|*|dj|)